src/main/java/org/dimdev/riftloader/RiftLoader.java

```diff
@@ -94,9 +109,22 @@ private void findMods(File modsDir) {
             throw new RuntimeException(e);
         }
 
-        // Load jar mods
+        modCount = modInfoMap.size() - modCount;
+        log.info("Loaded " + modCount + " mods");
+    }
+
+    /**
+     * Looks for Rift mods (jars containing a 'riftmod.json' at their root) in
+     * the 'modsDir' directory (creating it if it doesn't exist) and loads them
+     * into the 'modInfoMap'.
+     **/
+    private void findJarMods(File modsDir) {
+        this.findJarMods(modsDir, false);
+    }
+    private void findJarMods(File modsDir, Boolean createDir) {
+        int modCount = modInfoMap.size();
         log.info("Searching for mods in " + modsDir);
-        modsDir.mkdirs();
+        if (createDir) modsDir.mkdirs();
```

Comment:
```suggestion
        if (createDir) modsDir.mkdirs();
        else if (!(modsDir.exists() && modsDir.isDirectory())) return;
```
And this should be before `int modCount = modInfoMap.size();`